kaldb/pom.xml

```diff
@@ -351,7 +351,7 @@
         <dependency>
             <groupId>com.adobe.testing</groupId>
             <artifactId>s3mock-junit4</artifactId>
-            <version>2.1.24</version>
+            <version>2.2.3</version>
```

Comment:
Newest is 2.3.3, but that requires us to update several other dependencies including micrometer.